File: About_KV/nodejs/global.js
async function f(event) {
    event.console.log(await event.kv.getGlobal('exist'));

    event.console.log(await event.kv.getGlobal(['not-exist', 'exist', 'not-exist']))

    return {status: 200, body: 'done'};
}

exports.handler = f;

File: About_KV/lua/global.lua
local _M = {}

function _M.handler(event)
    event.console.log(json.encode((event.kv.get_global('exist'))))
    
    event.console.log(json.encode((event.kv.get_global({'not-exist', 'exist', 'not-exist'}))))

    return {status = 200, body = 'done'}
end

return _M